### DISCUSSION TITLE
Fishnet log entry line question

### DISCUSSION THREAD

**Discussion by ahbanks** [lichess-org/fishnet#171] [Q&A] [answered]
Just a curious question about the lines in the fishnet log that start with "><>".

These lines give values for knps (which are not delimited), batches, positions, and total nodes which are, but the separator is a ".".  Is it correct to interpret the period as a thousands separator?  IIRC that is a European standard, while here in the US we use a comma (",").  

Thanks!

**Comment by jim-perkins** [comment, marked as answer]
Yea it's a crazy world out there when we cant agree on a proper separator.  We're lucky we get anything accomplished :-)  LIchess is French.  The developer of fishnet, Niklas, gives his location as Germany https://github.com/niklasf.    https://en.wikipedia.org/wiki/Decimal_separator

Ya may as well get used to it.  I work at a large US manufacturer and in the past all the documentation for our manufacturing equipment was translated into our standards.  Now it comes as it was printed and ya gotta make sense of it.   

Completely unrelated to fishnet but slightly related to this question.  Why Do metric socket wrenches have the square head in english(imperial) dimensions ?  ie 1/4. 3/8/ , and 1/2 inch.  

But yes, the fishnet logs took me a little bit to get used to.

**Reply by ahbanks** [reply]
Good deal, that's what I suspected, but wasn't sure.  

As for the wrench - suspect it is a backwards-compatibility artifact that will likely be around forever.  Kind of like the QWERTY keyboard.
